--- origsrc/ecore-0.9.9.050/configure.in	2008-09-24 21:21:12.000000000 -0500
+++ src/ecore-0.9.9.050/configure.in	2008-12-24 00:24:18.896750000 -0600
@@ -94,6 +94,10 @@
                 fi
 		lt_no_undefined="-no-undefined"
 		;;
+	cygwin*)
+		AC_CHECK_HEADERS([dlfcn.h features.h langinfo.h locale.h netdb.h netinet/in.h sys/time.h sys/mman.h])
+		lt_no_undefined="-no-undefined"
+		;;
 	*)
 		AC_CHECK_HEADERS([dlfcn.h features.h langinfo.h locale.h netdb.h netinet/in.h sys/time.h sys/mman.h])
 		AC_CHECK_LIB(dl, dlopen, dlopen_libs=-ldl)
@@ -206,7 +210,7 @@
   )
 
   if test "x$have_iconv" != "xyes"; then
-    AC_CHECK_LIB(iconv, iconv,
+    AC_CHECK_LIB(iconv, libiconv,
       [
        iconv_libs="-liconv"
        have_iconv="yes"
--- origsrc/ecore-0.9.9.050/src/lib/ecore_con/Makefile.am	2008-08-17 02:42:41.000000000 -0500
+++ src/ecore-0.9.9.050/src/lib/ecore_con/Makefile.am	2008-12-24 00:35:11.709250000 -0600
@@ -23,7 +23,7 @@
 $(top_builddir)/src/lib/ecore/libecore.la \
 @SSL_LIBS@ @CURL_LIBS@
 
-libecore_con_la_LDFLAGS = -version-info @version_info@
+libecore_con_la_LDFLAGS = @lt_no_undefined@ -version-info @version_info@
 
 libecore_con_la_DEPENDENCIES = \
 $(top_builddir)/src/lib/ecore/libecore.la
--- origsrc/ecore-0.9.9.050/src/lib/ecore_config/Makefile.am	2008-08-17 02:42:41.000000000 -0500
+++ src/ecore-0.9.9.050/src/lib/ecore_config/Makefile.am	2008-12-24 00:35:32.506125000 -0600
@@ -27,7 +27,7 @@
 include_HEADERS = \
 Ecore_Config.h
 
-libecore_config_la_LDFLAGS = -version-info @version_info@
+libecore_config_la_LDFLAGS = @lt_no_undefined@ -version-info @version_info@
 
 #config_DATA = $(DB)
 #configdir = $(pkgdatadir)
--- origsrc/ecore-0.9.9.050/src/lib/ecore_desktop/Makefile.am	2008-08-17 02:42:41.000000000 -0500
+++ src/ecore-0.9.9.050/src/lib/ecore_desktop/Makefile.am	2008-12-24 00:35:41.099875000 -0600
@@ -27,7 +27,7 @@
 $(top_builddir)/src/lib/ecore/libecore.la \
 $(top_builddir)/src/lib/ecore_file/libecore_file.la
 
-libecore_desktop_la_LDFLAGS = -version-info @version_info@
+libecore_desktop_la_LDFLAGS = @lt_no_undefined@ -version-info @version_info@
 
 libecore_desktop_la_DEPENDENCIES = \
 $(top_builddir)/src/lib/ecore/libecore.la \
--- origsrc/ecore-0.9.9.050/src/lib/ecore_ipc/Makefile.am	2008-08-17 02:42:41.000000000 -0500
+++ src/ecore-0.9.9.050/src/lib/ecore_ipc/Makefile.am	2008-12-24 00:36:05.396750000 -0600
@@ -24,7 +24,7 @@
 $(top_builddir)/src/lib/ecore_con/libecore_con.la \
 @SSL_LIBS@
 
-libecore_ipc_la_LDFLAGS = -version-info @version_info@
+libecore_ipc_la_LDFLAGS = @lt_no_undefined@ -version-info @version_info@
 
 libecore_ipc_la_DEPENDENCIES = \
 $(top_builddir)/src/lib/ecore/libecore.la \
--- origsrc/ecore-0.9.9.050/src/lib/ecore_x/Makefile.am	2008-08-17 02:42:41.000000000 -0500
+++ src/ecore-0.9.9.050/src/lib/ecore_x/Makefile.am	2008-12-24 00:31:23.553000000 -0600
@@ -22,7 +22,7 @@
 
 libecore_x_la_LIBADD = $(DEP)
 
-libecore_x_la_LDFLAGS = -version-info @version_info@
+libecore_x_la_LDFLAGS = @lt_no_undefined@ -version-info @version_info@
 
 libecore_x_la_DEPENDENCIES = $(DEP)
 
