--- origsrc/ecore-0.9.9.063/src/lib/ecore_x/Makefile.am	2009-04-17 04:51:28.000000000 -0500
+++ src/ecore-0.9.9.063/src/lib/ecore_x/Makefile.am	2010-02-03 01:36:13.505951100 -0600
@@ -24,7 +24,7 @@ libecore_x_la_SOURCES =
 libecore_x_la_LIBADD = $(DEP) \
 @EINA_LIBS@
 
-libecore_x_la_LDFLAGS = -version-info @version_info@ @ecore_x_release_info@
+libecore_x_la_LDFLAGS = -no-undefined -version-info @version_info@ @ecore_x_release_info@
 
 libecore_x_la_DEPENDENCIES = $(DEP)
 
