--- origsrc/ecore-1.0.1/src/lib/ecore_x/Makefile.am	2011-04-19 01:50:23.000000000 -0500
+++ src/ecore-1.0.1/src/lib/ecore_x/Makefile.am	2011-11-03 19:41:03.133895600 -0500
@@ -18,7 +18,7 @@ lib_LTLIBRARIES = libecore_x.la
 libecore_x_la_SOURCES =
 
 libecore_x_la_LIBADD = $(DEP) @EINA_LIBS@
-libecore_x_la_LDFLAGS = -version-info @version_info@ @release_info@
+libecore_x_la_LDFLAGS = -no-undefined -version-info @version_info@ @release_info@
 libecore_x_la_DEPENDENCIES = $(DEP)
 
 includes_HEADERS = \
--- origsrc/ecore-1.7.0/src/modules/immodules/ibus/Makefile.am	2012-06-11 10:36:51.000000000 -0500
+++ src/ecore-1.7.0/src/modules/immodules/ibus/Makefile.am	2012-09-09 22:24:21.831043700 -0500
@@ -27,6 +27,7 @@ ibus_imcontext.c \
 ibus_imcontext.h
 
 ibus_la_LIBADD = \
+	 $(top_builddir)/src/lib/ecore_evas/libecore_evas.la \
 	 $(top_builddir)/src/lib/ecore_imf/libecore_imf.la \
 	 $(top_builddir)/src/lib/ecore_x/libecore_x.la \
 	@IBUS_LIBS@ \
