--- origsrc/ecore-1.0.1/src/lib/ecore_x/Makefile.am	2011-04-19 01:50:23.000000000 -0500
+++ src/ecore-1.0.1/src/lib/ecore_x/Makefile.am	2011-11-03 19:41:03.133895600 -0500
@@ -18,7 +18,7 @@ lib_LTLIBRARIES = libecore_x.la
 libecore_x_la_SOURCES =
 
 libecore_x_la_LIBADD = $(DEP) @EINA_LIBS@
-libecore_x_la_LDFLAGS = -version-info @version_info@ @release_info@
+libecore_x_la_LDFLAGS = -no-undefined -version-info @version_info@ @release_info@
 libecore_x_la_DEPENDENCIES = $(DEP)
 
 includes_HEADERS = \
